src/api/routes/company.js

```diff
@@ -258,23 +258,22 @@ export default (app) => {
     );
 
     /**
-     * Get the application of a company
+     * Get the state of a company with its id
      */
-    router.get("/:companyId/application",
+    router.get("/:companyId/state",
```

Comment:
Why did this route change? Don't we need to see the full applications when logged-in as an admin?

Reply:
We changed what was being returned by the route, instead of returning the whole application it is now only returning the state as we don't need the rest of the application's information.

Reply:
Actually, it could be better to return the whole application: makes this specific endpoint future-proof. I'll leave it up to you whether this changes or not. Eitherway good job.